WebDriverAgentTests/IntegrationTests/XCUIApplicationHelperTests.m

```diff
@@ -40,10 +40,22 @@ - (void)testQueringSpringboard
 
 - (void)testApplicationTree
 {
-  XCTAssertNotNil(self.testedApplication.fb_tree);
+  XCTAssertNotNil([self.testedApplication fb_tree:nil]);
+  
+  NSDictionary *applicationTree = [self.testedApplication fb_tree:[NSSet setWithArray:@[@"visible"]]];
```

Comment:
Please remove duplicated lines from this test

Reply:
removed